### PR TITLE
Improving hackathon page

### PR DESCRIPTION
**Pull request by kentonlam**
Apologies for git history and terrible HTML formatting.

- Buttons
- Project ideas
- Remove early bird

I intend to add a link to the showcase page, once that is merged.

### CODE REVIEW

File: content/competitions/hackathon-2020.html [hunk 1]
```diff
@@ -17,27 +17,66 @@ <h1 class="title is-3"><a href="https://www.facebook.com/events/2858423371101933
...
+
+            <h3>Projects</h3>
+            <p>
+                In the past people have built websites, desktop apps, games, hardware, and even robots. 
+                Some past projects are on our showcase page.
+                Still not sure what to do? Check out the ideas at the link below. 
+                If you have your own amazing idea, you can also add it and find teammates to work with!
+            </p>
+            <p>
+                <a class="button is-info" href="https://uqcs.org/showcase/">UQCS Showcase</a>
```
Comment (nicklambourne) [inline content/competitions/hackathon-2020.html:58]: I feel like this link should be relative

File: content/competitions/hackathon-2020.html [hunk 2]
```diff
@@ -17,27 +17,66 @@ <h1 class="title is-3"><a href="https://www.facebook.com/events/2858423371101933
...
+                <a href="https://www.facebook.com/events/2858423371101933/" class="button">Facebook event</a>
+            </p>
 
             <p>Howdy, y'all! We're happy to announce our biggest event of the year, the UQCS Hackathon, will be on
-                October 16th to 18th.</p>
+                <b>October 16th to 18th</b> at Room 301 of the Advanced Engineering Building.</p>
 
-            <p>If you aren't familiar with how the UQCS Hackathon works, it's pretty simple:</p>
+                <p>
+                    Form a team on Friday night, then spend two days building something cool—a game, a website, hardware,
```
Comment (nicklambourne) [inline content/competitions/hackathon-2020.html:29]: needs a space around the `-`

**Review by nicklambourne — CHANGES_REQUESTED** [review]
Is there a rationale to the choice of button colours? (i.e. which buttons have which colours). I feel like the one we particularly want to stand out at the moment is the ticket link.

Should a showcase page be merged before this PR so as to avoid the dead link issue?

File: content/competitions/hackathon-2020.html (hunk 1 above)
Comment (kentonlam) [reply content/competitions/hackathon-2020.html:58]: Sure.

**Review by nicklambourne — APPROVED** [review]

**Pull request merged**